app/views/checkins/new.html.erb

```
@@ -1,4 +1,4 @@
-<% unless @current_user.credentials.nil? %>
+<% unless not current_user.has_google_credential? %>
```

This SHOULD throw an error, the only variables you can pass to views are `@`. also `not` -> `!`